### PR TITLE
NFT Bonus Index

### PR DESCRIPTION
**Pull request by bonus-index**
When submitting a new or existing set details change, make sure you have followed the following steps:

### Adding New Set Details
- [ ]  Add a new file (or update existing file) into the `set-details` folder that is named as `[insert-your-set-address].setdetails.json` (i.e. 0x1494CA1F11D487c2bBe4543E90080AeBa4BA3C2b.setdetails.json). Make sure to provide the [checksum address](https://ethsum.netlify.app/) for your Set
- [ ] Verify the JSON follows the correct schema according to this [template](https://github.com/SetProtocol/uniswap-tokenlist/blob/main/set-details.schema.json). You can verify it by pasting the schema into this [schema validator](https://www.jsonschemavalidator.net) and checking if your input JSON matches properly. If you need to have multi-line JSON, you can use a HJSON converter to help with writing it and then paste in the final valid JSON spit out from it (see README for example)
- [ ] If adding brand new token, make sure to add it to the set.tokenlist.json as well following the correct schema from Uniswap: https://github.com/Uniswap/token-lists/blob/master/src/tokenlist.schema.json

### Updating Existing Set Details
- [ ] If updating existing set details, update the version number appropriately (see the JSON schema above for version numbering explanation).
- [ ] When generating a signature, make sure the version signed is for the one reflected in the version details for your entry.

### Signature (Required)

- [ ] Attach the proper signature. You can generate one by going to the page for your Set on TokenSets and signing in. Tap on your Set's name and follow the instructions provided (i.e. 0x21fbf0696d5e0aa2ef41a2b4ffb623bcaf070461d61cf7251c74161f82fec3a4370854bc0a34b3ab487c1bc021cd318c734c51ae29374f2beb0e6f2dd49b4bf41c):

[0x6112bc6995afa4560cc5aaee010c1bbcb53aa8bce16ac6e32b4bb7d38dae47a07f5d21047f00ec794ca3c4cf961b20d38ccd75a48fd8f32feb63171054c7163a1c]

### CODE REVIEW

File: set.tokenlist.json [hunk 1]
```diff
@@ -104,6 +104,14 @@
       "decimals": 18,
       "logoURI": "https://raw.githubusercontent.com/SetProtocol/uniswap-tokenlist/main/assets/tokens/apeszn.svg"
     }
```
Comment (asoong) [inline set.tokenlist.json:106]: Add a comma here

**Comment by bonus-index** [comment]
Hi Alex,

I need to perform some action, because the logo and instructions have not yet appeared

Sent with [ProtonMail](https://protonmail.com) Secure Email.

‐‐‐‐‐‐‐ Original Message ‐‐‐‐‐‐‐

File: set.tokenlist.json (hunk 1 above)
Comment (asoong) [reply set.tokenlist.json:106]: Bump!

**Comment by asoong** [comment]
Stale, feel free to re-open if still active

**Pull request closed without merging**